python/llm/example/CPU/Application/streaming/README.md

```diff
@@ -0,0 +1,40 @@
+# Low-Bit Streaming LLM using BigDL-LLM
+
+In this example, we apply [Streaming-LLM](https://github.com/mit-han-lab/streaming-llm/tree/main#efficient-streaming-language-models-with-attention-sinks) using BigDL-LLM, which can deploy low-bit(including INT8/INT5/INT4) LLMs for infinite-length inputs.
```

Comment:
In this example, we apply low-bit optimizations to [Streaming-LLM](https://github.com/mit-han-lab/streaming-llm/tree/main#efficient-streaming-language-models-with-attention-sinks) using BigDL-LLM, which can deploy low-bit (including FP4/INT4/FP8/INT8) LLMs for infinite-length inputs.